src/views/Home.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export const Home = () => {
+  return (
+    <>
+        
+        Hola mundo
+
+    </>
```

Comment:
en este caso al ser un texto debe ir en un tag 
```suggestion
    <h2>  
        Hola mundo
    </h2>
```